_pytest/main.py

```diff
@@ -177,6 +190,15 @@ def pytest_ignore_collect(path, config):
     if py.path.local(path) in ignore_paths:
         return True
 
+    allow_in_venv = config.getoption("collect_in_virtualenv")
+    if _in_venv(path) and not allow_in_venv:
```

Comment:
We probably want to make this check only if the user did not explicitly passed the path into the command line, like what we do to while collecting python test files in [`python.py`](https://github.com/pytest-dev/pytest/blob/master/_pytest/python.py#L149).

IOW, `pytest .env` should collect everything under `.env` even if `.env` is a virtual environment directory.